tests/integration/test_reinstall.py

```diff
@@ -244,7 +244,7 @@ def test_rose_warning(one_src, one_run, capsys, interactive, monkeypatch):
 
     # reinstall (no rose-suite.conf file)
     reinstall_cli(opts=ReInstallOptions(), args=one_run.id)
-    assert rose_message not in capsys.readouterr().err
+    assert rose_message not in capsys.readouterr().out
```

Comment:
This was correct before, it is tested in line 252.

Reply:
```suggestion
    assert rose_message not in capsys.readouterr().err
```